docs/WritingTests.asciidoc

```diff
@@ -123,10 +280,15 @@ sub run() {
     # capture a screenshot and compare with needle 'test-zypper_in'
     assert_screen 'test-zypper_in';
 }
+
+1;
```

Comment:
Please leave this out. The example above is not a complete file content, just the implementation of one method